src/main/java/core/basesyntax/service/DataConverter.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface DataConverter {
+    /**
```

Comment:
Don't begin interface implementation with an empty line. Please remove the redundant empty line at the start of the interface definition.